system-test/pubsub.ts

```diff
@@ -148,7 +150,7 @@ describe('pubsub', () => {
   describe('Topic', () => {
     it('should be listed', async () => {
       const [topics] = await pubsub.getTopics();
-      const results = topics.filter(topic => {
+      const results = topics.filter((topic: Topic) => {
```

Comment:
That doesn't look right.  What's the type of `topics` here?  Shouldn't need to declare this type explicitly.

Reply:
linter fix it automatically. I will make it back.